framework/src/media/utils/MediaUtils.cpp

```diff
@@ -68,10 +72,257 @@ audio_type_t getAudioTypeFromPath(std::string datapath)
 	} else if ((extension.compare("") == 0) || (extension.compare("pcm") == 0) || (extension.compare("raw") == 0)) {
 		medvdbg("audio type : pcm\n");
 		return AUDIO_TYPE_PCM;
+	} else if (extension.compare("wav") == 0) {
+		medvdbg("audio type : wav\n");
+ 		return AUDIO_TYPE_WAVE;
 	} else {
 		medvdbg("audio type : unknown\n");
 		return AUDIO_TYPE_INVALID;
 	}
 }
+
+bool mp3_header_parsing(unsigned char *header, unsigned int *channel, unsigned int *sample_rate, audio_format_type_t *pcmFormat)
+{
+	*pcmFormat = AUDIO_FORMAT_TYPE_S16_LE;
+	
+	unsigned char bit;
+	unsigned int mpeg_version;	
+	bit = header[1];
+	bit >>= 3;
+	bit &= (unsigned char)3;
+
+	switch (bit) {
+	case 0 :
+		mpeg_version = 2;
+		break;
+	case 2 :
+		mpeg_version = 1;
+		break;
+	case 3 :
+		mpeg_version = 0;
+		break;
+	default :
+		medvdbg("mpeg version is wrong\n");
+		return false;
+	}
+
+	bit = header[2];
+	bit >>= 2;
+	bit &= 3;
+	
+	switch (bit) {
+	case 0 :
+		*sample_rate = 44100;
+		break;
+	case 1 :
+		*sample_rate = 48000;
+		break;
+	case 2 :
+		*sample_rate = 32000;
+		break;
+	default :
+		medvdbg("sample rate is wrong\n");
+		return false;
+	}
+
+	*sample_rate >>= mpeg_version;
+
+	bit = header[3];
+	bit >>= 6;
+	if (bit <= 2) {
+		*channel = 2;
+	} else {
+		*channel = 1;
+	}
+
+	return true;
+}
+
+bool aac_header_parsing(unsigned char *header, unsigned int *channel, unsigned int *sample_rate, audio_format_type_t *pcmFormat)
```

Comment:
both of mp3, aac case, if we don't use pcmformat when parse header, we don't have to add them as a param.
But why don't you check with SRCN?